README.md

```diff
@@ -73,7 +73,7 @@
         ```sh
         serve ./dist
         ```
-    1. Open the app at http://localhost:5000/index-self-contained.html (Note the different path)
+    1. Open the app at http://localhost:5000/index.html
 
 ## Working with local dependencies
 
```

Comment:
in the next paragraph, the sentence is incomplete:

"...and the impact (missing: 'becomes visible' or similar) in your application immediately."

Reply:
Thanks. I changed it to "and **see** the impact in your application immediately".

Reply:
👍